### ISSUE TITLE
Feature Request: Folder level bzlgenrc overrides

### ISSUE THREAD

**Issue by codebreach** [Evertz/bzlgen#61]
We want different `ng_module` and visibility on a per folder level `projects/a` vs `projects/b`

As far as I can tell, there can only be one `.bzlgenrc` at the root level.

So the question is - what would it take to support folder level overrides?

I (or someone from our team) can take this up if you need help. We are planning on using bzlgen to help migrate and manage build files in our large (200k LOC) angular codebase.

**Comment by mattem** [comment]
Correct, currently only a root level `.bzlgenrc` file is supported. 

The rc file and all flags are resolved within [`Flags#setupAndParseArgs`](https://github.com/Evertz/bzlgen/blob/master/src/flags.ts#L190). If the files were placed in the folder they were overriding, this would then have to search up from the given `<path>` until it found any folder level rc files, until it reached the root, and then merged them. Alternative to searching for overrides is perhaps listing them in the root, and loading them in.

Another approach would be to have a mapping of folder to override flags, as a prefix to the value perhaps, `--default_visibility=projects/a=//project/b:__pkg__` which would set the default visibility to `//project/b:__subpkg__` when generating in `projects/a`.
